app/models/email.rb

```diff
@@ -10,14 +18,21 @@ class Email
   attribute :subject, :type => String
   attribute :content, :type => String
 
-  validates_presence_of :hide_recipients, :recipients, :reply_to, :subject, :content
+  validates_presence_of :recipients, :reply_to, :subject, :content
+  validates_inclusion_of :hide_recipients, in: [true, false]
 
   def initialize(attributes = {})
     attributes.each do |name, value|
       send("#{name}=", value)
     end
   end
 
+  def validate_attributes(attributes = [])
```

Comment:
I couldnt find a test for this.
Not really sure what it does too. Why `validates_presence_of`?
Few lines above you also use `validates_inclusion_of`